runtime/bcutil/defineclass.c

```diff
@@ -511,9 +511,14 @@ internalLoadROMClass(J9VMThread * vmThread, J9LoadROMClassData *loadData, J9Tran
 		translationFlags |= stripFlags;
 	}
 
-	if ((0 == (loadData->options & J9_FINDCLASS_FLAG_UNSAFE)) &&
-		(0 != (vm->runtimeFlags & J9_RUNTIME_VERIFY))) {
-		translationFlags |= BCT_StaticVerification;
+	/* Enable the static verification on the unsafe classes (only for Java 8 & 11) */
+	if (J9_ARE_ANY_BITS_SET(vm->runtimeFlags,J9_RUNTIME_VERIFY)) {
+		if (J9_ARE_NO_BITS_SET(loadData->options, J9_FINDCLASS_FLAG_UNSAFE)
+		|| (J2SE_VERSION(vm) == J2SE_18)
+		|| (J2SE_VERSION(vm) == J2SE_V11)
+		) {
+			translationFlags |= BCT_StaticVerification;
+		}
```

Comment:
Ideally, this becomes:
```suggestion
	if (J9_ARE_ANY_BITS_SET(vm->runtimeFlags,J9_RUNTIME_VERIFY)) {
		translationFlags |= BCT_StaticVerification;
	}
```

I'd like to verify as much of the Unsafe class as possible.

Reply:
Removed the check on Java 8  & 11. Will double-check in the personal builds.